ncat/ncat_main.c

```diff
@@ -280,6 +280,7 @@ int main(int argc, char *argv[])
         {"lua-exec",        required_argument,  NULL,         0},
         {"lua-exec-internal",required_argument, NULL,         0},
 #endif
+        {"use-delimiter",   required_argument,  NULL,         'b'},
```

Comment:
Please report to me whether traditional netcat or openbsd netcat uses the `-b` short option and what they use it for. This seems to be a new functionality that is not part of either one, so it does not need a short option.